guides/v2.1/cloud/reference/docker-config.md

```diff
@@ -0,0 +1,73 @@
+---
+group: cloud
+title: Configure Docker
+version: 2.1
+github_link: cloud/reference/docker-config.md
+functional_areas:
+  - Cloud
+  - Setup
+  - Configuration
+---
+
+{{site.data.var.ece}} provides a Docker environment option for those who use their local environment for develop and testing, or for automation tasks. There are three, essential parts to launching the Magento Cloud Docker environment: a Magento 2 template, Docker Compose, and Magento Cloud Tools.
+
+The [Magento Cloud Docker repository](https://github.com/magento/magento-cloud-docker){:target="\_blank"} contains build information for the following [Docker hub](https://hub.docker.com/r/magento/){:target="\_blank"} images, including the following:
+
+-  PHP: magento/magento-cloud-docker-php
+    -  PHP-CLI - version 7 and later
+    -  PHP-FPM - version 7 and later
+-  NGINX:  magento/magento-cloud-docker-nginx
+
+The Cloud tools provides a `docker:build` command to generate the Docker Compose configuration. Also, you can specify a version using one of the following options:
```

Comment:
Magento Cloud Tools provides .... 

Is Magento Cloud Tools a package, or a repository?

If so, "The Magento Cloud Tools" package provides ----   sounds better because of the awkward subject verb agreement with just "Cloud Tools"

Reply:
Magento Cloud Tools is Magento Cloud, almost. Cloud Tools is my compromise instead of using `ece-tools`. I haven't noticed whether this has been addressed by Marketing, yet. I try not to say repository unless I need them to look at the Github stuff. This is definitely up for debate.